src/views/LandingPage.vue

```diff
@@ -94,34 +94,39 @@
               <p class="text-3xl font-bold">
                 {{ cardBack.heading }}
               </p>
-              <p>{{ cardBack.text }}</p>
+              <p class="text-sm">
+                {{ cardBack.text }}
+              </p>
             </div>
 
-            
           </div>
         </div>
       </div>
     </div>
-    <FadeTeaser>
-      <div class="absolute top-20 xl:py-20 left-0 right-0">
-        <div class="flex flex-col text-center">
-          <p class="text-6xl text-teal-500 font-bold">Be part of the game</p>
-          <div class="text-black text-3xl pt-4">
-            <p>Crowd Control is a growing community based on open source and</p>
-            <p>collaboration. Learn the game and give us your review.</p>
+    <div class="h-[55rem] xl:h-[60rem] overflow-clip">
```

Comment:
Maybe that's something that should be done in the fade teaser itself. Would also be a good idea to incorporate this directly into my PR, since fade teaser behaviour is changed there.

Reply:
The fade teaser is used in different situations, sometimes as the whole background of a page (learn and other pages) and sometimes only as a fragment (landing page)